src/systems/systemDetails.component.tsx

```diff
@@ -184,16 +186,21 @@ function SystemDetails(props: SystemDetailsProps) {
               {system.description ?? 'None'}
             </Typography>
           </Grid>
-          <Grid item>
-            <Typography variant="h6" sx={{ paddingTop: 4 }}>
-              Items
-            </Typography>
-          </Grid>
-          <Grid item>
-            <Divider />
-          </Grid>
-          <Grid item>
-            <SystemItemsTable system={system} type="normal" />
+          <Grid item container sx={{ marginTop: 2, display: 'inline-block' }}>
+            <TabView
+              ariaLabelPrefix="systems page"
+              defaultTab="Items"
+              galleryEntityId={system.id}
+              attachmentsEntityId={system.id}
```

Comment:
Does `galleryEntityID `and `attachmentsEntityId` refer to the id of the actual "gallery/attachment object" or their parent/foreign key (in this case system.id).

If its the former, I assume system.id is just a placeholder until they get implemented, but if its the latter it would make sense to combine parameters.

Reply:
The tabview is going to be used in other places. So the gallery and attachments needs to be optional. That is why it has been implemented like this.